### PR TITLE
✨ Add Rating component

### LINKED ISSUE #9: Movie details: Add rating element
To do:

- [ ] Add rating element that displays the user rating as a number (e.g. 4.0)
- [ ] Add rating element that displays the user rating as scale (stars, each star equalling 2 points)
Screenshot 2021-05-25 at 12 16 17

### PR DESCRIPTION
**Pull request by lmachens**
Fixes #9 and is required by #4.

I added more CSS variables to control gradients in SVGs.

### CODE REVIEW

**Review by lajanzen — APPROVED** [review]
lgtm 🚀

**Review by fgottesmann — APPROVED** [review]
Looks good to me

**Review by roabram — APPROVED** [review]

**Review by Dev-dfm — COMMENTED** [review]
lgtm 🚀

**Review by tmh9x — APPROVED** [review]

File: src/app/components/Rating/Star.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const UNCHECKED_PATH =
+  'M13.55,7.061A.993.993,0,0,0,13,5.366L9.645,4.879a.438.438,0,0,1-.33-.24L7.816,1.6a.993.993,0,0,0-1.781,0l-1.5,3.039a.438.438,0,0,1-.33.24L.851,5.367A.993.993,0,0,0,.3,7.061L2.727,9.426a.439.439,0,0,1,.126.388l-.573,3.34A.993.993,0,0,0,3.722,14.2l3-1.577a.439.439,0,0,1,.408,0l3,1.577a.993.993,0,0,0,1.441-1.047L11,9.814a.439.439,0,0,1,.126-.388Zm-3.1,2.847.573,3.34a.439.439,0,0,1-.636.463l-3-1.577a.993.993,0,0,0-.925,0l-3,1.577a.439.439,0,0,1-.636-.463L3.4,9.908a.993.993,0,0,0-.286-.879L.688,6.664a.439.439,0,0,1,.243-.748l3.353-.487a.992.992,0,0,0,.748-.543l1.5-3.039a.439.439,0,0,1,.787,0l1.5,3.039a.991.991,0,0,0,.748.543l3.354.487a.439.439,0,0,1,.243.748L10.736,9.029A.993.993,0,0,0,10.45,9.908Z';
+
+function Star({ checked }: StarProps): JSX.Element {
+  return (
+    <svg width="13.85" height="13.246" viewBox="0 0 13.85 13.246">
+      <defs>
+        <linearGradient
+          id="star-linear-gradient"
```
Comment (phlgr) [inline src/app/components/Rating/Star.tsx:17]: This can be removed

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/Rating/Star.tsx (hunk 1 above)
Comment (lmachens) [reply src/app/components/Rating/Star.tsx:17]: No, it is required by the `fill`:
`fill="url(#star-linear-gradient)"`

**Review by phlgr — APPROVED** [review]
🚀 🥇

**Pull request merged**